release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -239,6 +239,18 @@ You must still use the `performance-addon-operator-must-gather` image when runni
 [id="ocp-4-11-nodes"]
 === Nodes
 
+[id="ocp-4-11-nodes-self-node-remediation"]
+==== Self Node Remediation Operator replaces the Poison Pill Operator
+{product-title} 4.11 introduces the Self Node Remediation Operator that replaces the Poison Pill Operator.
+
+The Self Node Remediation Operator provides the following enhancements:
+
+* Introduces separate remediation templates based on the remediation strategy
+* Captures last error message if remediation is unsuccessful
+* Provides metrics for Operator's configuration parameters
```

Comment:
I'd consider changing the phrasing a bit.
IIUC there no new metrics that are exposed the change is mainly that we use min/max values for that metrics and make them more "human readable" by eliminating some unused time unit options.
Maybe `Improving` over `Providing`  ?

Reply:
I agree with Michael